### PR TITLE
New package: hors

### PR DESCRIPTION
**Pull request by kcubeterm**
instant coding answers via the command line written in rust similar to howdoi

### CODE REVIEW

File: packages/hors/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	termux_setup_rust
+
+	cd "$TERMUX_PKG_SRCDIR"
+        Cargo install hors
+}
+
+termux_step_make_install() {
+	install -Dm700 -t "$TERMUX_PREFIX"/bin "$HOME"/.cargo/bin/hors
+	install -Dm600 -t "$TERMUX_PREFIX"/share/doc/hors/ "$TERMUX_PKG_SRCDIR"/README*
+}
```
Comment (xeffyr) [inline packages/hors/build.sh:18]: You shouldn't use `cargo` in this way.

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/hors/build.sh (hunk 1 above)
Comment (xeffyr) [reply packages/hors/build.sh:18]: Normally, you need to add `TERMUX_PKG_BUILD_IN_SRC=true` and remove function overrides. See how other rust-based packages are built.

Comment (kcubeterm) [reply packages/hors/build.sh]: Oh ,okay,

Comment (xeffyr) [reply packages/hors/build.sh]: Remove function steps if Cargo.toml is located at project root.

Comment (kcubeterm) [reply packages/hors/build.sh]: yea, it is, do you mean, Termux_step_make function

**Comment by kcubeterm** [comment]
This same build file gives me error locally.

File: packages/hors/build.sh (hunk 1 above)
Comment (xeffyr) [reply packages/hors/build.sh]: Few things to look at:
* https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_make_install.sh - You shouldn't override this if not necessary.
* https://github.com/termux/termux-packages/blob/master/scripts/lint-packages.sh#L25 - you have wrong license name. Should be `GPL-3.0` exactly.

Fixed myself.

Comment (kcubeterm) [reply packages/hors/build.sh]: Got it, thankyou,

**Comment by kcubeterm** [comment]
> This same build file gives me error locally.

Because you have wrong licence name lol

**Comment by xeffyr** [comment]
> Because you have wrong licence name lol

Because you don't know SPDX license identifiers lol

**Comment by kcubeterm** [comment]
This much warning .

**Review by xeffyr — APPROVED** [review]

**Pull request merged**